### ISSUE TITLE
(Russian) Cannot create new Trezor wallet due to UI 

### ISSUE THREAD

**Issue by jim618** [bitcoin-solutions/multibit-hd#837]
The Russian translations push the buttons off the screen for the Trezor create new wallet operation. You cannot complete the wizard.

Screenshot:
![screen shot 2016-01-12 at 15 03 31](https://cloud.githubusercontent.com/assets/919427/12267085/47beaaca-b93e-11e5-91e3-b6939e145744.png)

**Comment by jim618** [comment]
Suggest we also check the regular create wallet in russian as it might have a similar problem

**Comment by gary-rowe** [comment]
Looks the same as the German one recently (see #773). We need to add a proper wrapper as in other views.

**Comment by gary-rowe** [comment]
@jim618 can you run through the new FEST tests. You'll see that the Trezor and KeepKey devices have had their UI's translated in error. They are supposed to be partitioned away to avoid being included.

**Comment by gary-rowe** [comment]
I've adjusted the Trezor and KeepKey components to use HTML wrapping which should allow for correct display over a variety of languages:

<img width="1012" alt="screen shot 2016-01-31 at 16 19 38" src="https://cloud.githubusercontent.com/assets/342133/12703288/6b678200-c837-11e5-91ea-dc75a5b9516a.png">

Unfortunately, I cannot get the title to show in Russian without a small amount of clipping which is annoying.

Please run this up with different languages and explore various screens with lengthy titles to ensure that there is no regression.

Ready for review and close.

**Comment by jim618** [comment]
I have created in Russian a Trezor wallet and a soft wallet.
All screens looked fine apart from this one:

![screen shot 2016-02-01 at 10 49 41](https://cloud.githubusercontent.com/assets/919427/12715468/683ccc16-c8d2-11e5-94a9-eae0f42d2f2d.png)

I could still navigate off the screen so it's not fatal.

**Comment by jim618** [comment]
Sorry - meant a KeepKey wallet

**Comment by jim618** [comment]
Just created a Trezor wallet in German and it all looked good.

**Comment by gary-rowe** [comment]
This also applied to the Trezor screen (they use the same underlying panel view). I've increased the use of HTML wrapping in labels, especially in status labels so they should wrap much more predictably now across a range of languages (including RTL/LTR).

Here's an example of what to expect:
<img width="1012" alt="screen shot 2016-02-01 at 11 27 52" src="https://cloud.githubusercontent.com/assets/342133/12716290/2990ed08-c8d7-11e5-939c-62900b78aebe.png">

Ready for review and close.

PS I also hardened the handling of a failed device after switching languages and wallet types. Kind of a corner case but the UX was no response to attach/detach of device in the Unlock screen.
